scripts/export-test-result.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { getSecretKey, executeCommand, printLog, logAndExit } = require('./utils')
+
+async function main() {
+  await exportTestResult('unit', ['test-report/unit/', 'test-report/unit-bs/'])
+  await exportTestResult('e2e', ['test-report/e2e/', 'test-report/e2e-bs/'])
+}
+
+async function exportTestResult(type, folders) {
+  const ddCiApiKey = (await getSecretKey('ci.browser-sdk.datadog_ci_api_key')).trim()
```

Comment:
Maybe `getSecretKey` could `trim` the key itself (for all secret keys)?

Reply:
I say, yes!